tests/wpt/mozilla/tests/mozilla/interfaces.js

```diff
@@ -45,9 +45,9 @@ function test_interfaces(interfaceNamesInGlobalScope) {
       "Uint32Array",
       "Uint8Array",
       "Uint8ClampedArray",
-      "Uint8ClampedArray",
       "WeakMap",
       "WeakSet",
+      "WebKitCSSMatrix",
```

Comment:
Whoops; this belongs in interfaces.html instead, and with a lowercase K.

Reply:
Whoooops!! Fixed it. It's `WebKitCSSMatrix` anyway though, as WebKit people somehow decided to use uppercase K here.